docker/cygnus-ngsi/agent.conf

```diff
@@ -167,6 +166,30 @@ cygnus-ngsi.sinks.postgresql-sink.postgresql_password =
 #cygnus-ngsi.sinks.postgresql-sink.backend.enable_cache = false
 
 
+# NGSIOrionSink configuration
+cygnus-ngsi.sinks.orion-sink.type = com.telefonica.iot.cygnus.sinks.NGSIOrionSink
+cygnus-ngsi.sinks.orion-sink.channel = orion-channel
+# cygnus-ngsi.sinks.orion-sink.enable_encoding = false
```

Comment:
If this is commented on purpose a comment line explainin why should be added.

Reply:
fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1529/commits/a6681b7d4caffc733caeaa45915351e9c3f4a6e3